### ISSUE TITLE
Potential to add nexus/spell removal while on high risk worlds?

### ISSUE THREAD

**Issue by Frooby91** [CalumMagee/wildernesswarnings#2]
I was informed that the rules have changed a bit since the plugins inception. The black chin TP is able to be removed like it is with custom menu swaps already. I was hoping this feature could be added to the portal nexus/manually casting spells as well? If those are not possible an addition(new?) plugin that simply adds a skull over the portal nexus when on high risk worlds could also work.

I ask for these changes because I have seen multiple irons lose 100+ hours accidentally. 

Basically, I would like the black chins tp, manually casting ice plat/cemetery etc & portal nexus options to be removed/changed while on high risk worlds like everything else is. Thank you.

**Comment by LuxOG** [comment]
I can remove the black chin tp and manual casting when I get around to it. The portal nexus uses a different implementation and would require an entirely different approach so I am not interested in doing that right now.

**Comment by Frooby91** [comment]
sounds good. thank you.

**Comment by LuxOG** [comment]
I am adding the manual spellbook cast hides and the portal nexus hides as I found a workaround for them. However, the black chins hide would require an annoying rewrite as the custom menu swaps plugin that portion is based off of got updated when Jagex allowed dynamic menu swaps for the max cape, so I'm leaving that one.
